src/core/Table/Table.tsx

```diff
@@ -259,6 +274,12 @@ export const Table = <
       case TableActions.setFilter:
         onFilterHandler(newState, action, previousState, instance);
         break;
+      case TableActions.toggleRowSelected:
```

Comment:
tests would be great..

Reply:
Added to check that this bug is not reproducible.